src/common/components/select-repository-list/index.js

```diff
@@ -125,6 +125,25 @@ export class SelectRepositoryListComponent extends Component {
     return renderedRepos;
   }
 
+  renderRepoAmount() {
+    const { ids, isFetching } = this.props.repositories;
+    const { selectedRepositories } = this.props;
+
+    if (isFetching && ids.length === 0) {
+      return (
+        <span>Calculating your total repositories&hellip;</span>
```

Comment:
I don't think we should be showing it at all.

First of all, it's not in [spec](https://docs.google.com/document/d/1koTwI3cRr2S2vj5zHKKbR7aWxLRfShDVq2gHQ29cDyU/edit#), secondly showing it is contrary to delaying showing the loading spinner. We delay showing the spinner for 2 seconds (not to show immediately that we are loading), but this text would basically be a "loading..." text that appears for a very short time.

The spec doesn't specify it in detail, but I suppose that before we load anything (so we don't know how many repos there are or how many pages there are) we shouldn't show any text at all.